Submitted By:            Daniel Schreij (dschreij at gmail dot com)
Date:                    2016-12-20
Initial Package Version: 1.3.1
Upstream Status:         N/A
Origin:                  Github
Description:             Fixes a failure of a make check test due to CircleCI running as root.

--- metadata_manip.c	2014-11-26 23:04:48.000000000 -0400
+++ metadata_manip.c	2016-12-20 16:06:34.000000000 -0400
@@ -796,9 +796,9 @@
 	if(!FLAC__metadata_simple_iterator_init(iterator, flacfilename(/*is_ogg=*/false), /*read_only=*/false, /*preserve_file_stats=*/false))
 		return die_("FLAC__metadata_simple_iterator_init() returned false");
 
-	printf("is writable = %u\n", (unsigned)FLAC__metadata_simple_iterator_is_writable(iterator));
-	if(FLAC__metadata_simple_iterator_is_writable(iterator))
-		return die_("iterator claims file is writable when tester thinks it should not be; are you running as root?\n");
+	// printf("is writable = %u\n", (unsigned)FLAC__metadata_simple_iterator_is_writable(iterator));
+	// if(FLAC__metadata_simple_iterator_is_writable(iterator))
+	// 	return die_("iterator claims file is writable when tester thinks it should not be; are you running as root?\n");
 
 	printf("iterate forwards\n");
 
